server/app/controllers/dev/seeding/SampleQuestionDefinitions.java

```diff
@@ -121,7 +146,36 @@ public final class SampleQuestionDefinitions {
               .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
               .build());
 
-  public static final NumberQuestionDefinition NUMBER_QUESTION_DEFINITION =
+  static final QuestionDefinition NAME_QUESTION_DEFINITION =
+      new NameQuestionDefinition(
+          QuestionDefinitionConfig.builder()
+              .setName("Name")
+              .setDescription("The applicant's name")
+              .setQuestionText(
+                  LocalizedStrings.of(
+                      ImmutableMap.of(
+                          Lang.forCode("am").toLocale(),
+                          "ስም (የመጀመሪያ ስም እና የመጨረሻ ስም አህጽሮት ይሆናል)",
+                          Lang.forCode("ko").toLocale(),
+                          "성함 (이름 및 성의 경우 이니셜도 괜찮음)",
+                          Lang.forCode("so").toLocale(),
+                          "Magaca (magaca koowaad iyo kan dambe okay)",
+                          Lang.forCode("lo").toLocale(),
+                          "ຊື່ (ນາມສະກຸນ ແລະ ຕົວອັກສອນທຳອິດຂອງນາມສະກຸນແມ່ນຖືກຕ້ອງ)",
+                          Lang.forCode("tl").toLocale(),
+                          "Pangalan (unang pangalan at ang unang titik ng apilyedo ay okay)",
+                          Lang.forCode("vi").toLocale(),
+                          "Tên (tên và họ viết tắt đều được)",
+                          Lang.forCode("en-US").toLocale(),
+                          "Please enter your first and last name",
+                          Lang.forCode("es-US").toLocale(),
+                          "Nombre (nombre y la inicial del apellido está bien)",
+                          Lang.forCode("zh-TW").toLocale(),
+                          "姓名（名字和姓氏第一個字母便可）")))
+              .setQuestionHelpText(LocalizedStrings.withDefaultValue("help text"))
+              .build());
```

Comment:
Above moved from `DatabaseSeedTask`, as a variable.

Both the minimal and comprehensive sample programs use this question; previously it was just taken from `DatabaseSeedTask` but we are trying to separate these two out.